conans/test/assets/files/autotools/Makefile.in

```diff
@@ -0,0 +1,746 @@
+# Makefile.in generated by automake 1.16.3 from Makefile.am.
```

Comment:
I think it doesn't make sense to have an almost 1 thousand lines auto generated file that we cannot understand and that contains thing like ``pdf: pdf-am`` in our codebase. This file probably should be generated in the test, a very minimal file should be used, or other alternatives considered.

Reply:
yes, that's the plan. I checked in generated files, as our CI lacks some tools used to generate:
- aclocal
- automake
- autoconf

I need @czoido help to install them in subsystems.
meanwhile, I can test with locally generated files.